### PR TITLE
Capacity parsers - add README

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue

Documentation and methodology around capacity parsers is missing

## Description

README file with necessary info to enable contributions from the community.
Include info on:
- when to do the update
- how to update capacities
- technical requirements for new capacity parser

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: electricitymap/contrib/capacity_parsers/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- [Capacity source](#capacity_sources)
+- [Capacity update process](#capacity_update_process)
+
+  - [When should capacity data be updated?](#when_to_update)
+  - [Format of the capacity configuration](#format)
+  - [Opening a PR](#PR)
+  - [The zone capacity can be updated automatically](#automatic_update)
+  - [The zone capacity is updated manually](#manual_update)
+
+- [Technical requirements for adding a new data source](#technical_requirements)
```
Comment (madsnedergaard) [inline electricitymap/contrib/capacity_parsers/README.md:15]: Fixes ToC formatting by removing blank lines:

```suggestion
- [Capacity update process](#capacity_update_process)
  - [When should capacity data be updated?](#when_to_update)
  - [Format of the capacity configuration](#format)
  - [Opening a PR](#PR)
  - [The zone capacity can be updated automatically](#automatic_update)
  - [The zone capacity is updated manually](#manual_update)
- [Technical requirements for adding a new data source](#technical_requirements)
```

Before:
Screenshot 2023-11-14 at 13 33 29

After:
Screenshot 2023-11-14 at 13 33 21

File: electricitymap/contrib/capacity_parsers/README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+One way to perform outlier detection is to check that each incoming data point is not higher than the installed capacity for a given mode. The power output cannot be above the energy input as the efficiency of power plants is always below 100%. In other words, for a given zone and a given mode, the power production for that mode at any given time cannot exceed the installed capacity for that mode.
+
+> **Example**
+> In 2023, the wind capacity in **DK-DK1** was 5233 MW. The average wind production in the first 3 quarters of 2023 was 1455 MW.
+
+The goal here is to validate each incoming production parser event by comparing each mode production against the installed capacity available in the zone configuration. If a mode production is higher than the installed capacity, the data point will be flagged as an outlier and will be corrected by our data pipelines.
+
+To achieve this goal, we need robust and consistent capacity data. We also need to be able to capture the evolution of capacity data over time. As renewable capacity increases in most zones, this means that the power production will also increase.
```
Comment (madsnedergaard) [inline electricitymap/contrib/capacity_parsers/README.md:31]: Maybe we should just reference the wiki for this section? Think it's less important to also have it here :)

**Review by madsnedergaard — APPROVED** [review]
Beautiful, thanks for this very extensive documentation 😍 

I'll add the wiki page to the sidebar so it's findable :)

Screenshot 2023-11-14 at 13 41 34

File: electricitymap/contrib/capacity_parsers/README.md (hunk 2 above)
Comment (mathilde-daugy) [reply electricitymap/contrib/capacity_parsers/README.md:29]: will do :)

**Pull request merged**